requirements.txt

```diff
@@ -1,26 +1,70 @@
 amqp==5.0.6
 asgiref==3.3.1
+atomicwrites==1.4.0
+attrs==21.2.0
+autobahn==21.3.1
+Automat==20.2.0
 billiard==3.6.4.0
+carehare==1.0.1
 celery==5.0.5
 certifi==2020.12.5
+cffi==1.14.5
 chardet==4.0.0
 click==7.1.2
 click-didyoumean==0.0.3
 click-plugins==1.1.1
 click-repl==0.1.6
+colorama==0.4.4
+constantly==15.1.0
+cryptography==3.4.7
+daphne==3.0.2
 Django==3.1.7
+django-appconf==1.0.4
+django-environ==0.4.5
+django-imagekit==4.0.2
+factory-boy==3.2.0
+Faker==8.9.0
+flake8==3.9.2
 gunicorn==20.1.0
+hyperlink==21.0.0
 idna==2.10
+incremental==21.3.0
+iniconfig==1.1.1
 kombu==5.1.0
+mccabe==0.6.1
+msgpack==1.0.2
 oauthlib==3.1.0
+packaging==20.9
+pamqp==3.0.1
+pilkit==2.0
+Pillow==8.2.0
+pluggy==0.13.1
 prompt-toolkit==3.0.18
 protobuf==3.15.5
 psycopg2==2.8.6
+py==1.10.0
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pycodestyle==2.7.0
+pycparser==2.20
+pyflakes==2.3.1
+pyOpenSSL==20.0.1
+pyparsing==2.4.7
+pytest==6.2.4
+pytest-django==4.4.0
+python-dateutil==2.8.1
 pytz==2021.1
 requests==2.25.1
 requests-oauthlib==1.3.0
+service-identity==21.1.0
 six==1.15.0
 sqlparse==0.4.1
+text-unidecode==1.3
+toml==0.10.2
+Twisted==21.2.0
+twisted-iocpsupport==1.0.1
```

Comment:
Maybe you need to remove twisted-iocpsupport==1.0.1 because I can't start project with this package.